services/graphql-server/directives/common/gql.go

```diff
@@ -78,8 +78,11 @@ func createGqlClient(url string) *gqlclient.Client {
 }
 
 func createGqlRequest(s server.ServerContext, gqlParams gqlParams, rp graphql.ResolveParams) (*gqlclient.Request, error) {
-	query := utils.ResolveParamsToSDLQuery(gqlParams.queryName, rp, gqlParams.args)
-	request := gqlclient.NewRequest(query)
+	requestConfig := utils.ResolveParamsToSDLRequest(gqlParams.queryName, rp, gqlParams.args)
```

Comment:
It's odd that we pass in the "raw" args before the variable insertion. `ResolveParamsToSDLRequest` should have one job, which is taking the queryName&ast and returning the SDL (with extra stuff like variables fragments etc if needed)